src/components/portal/inventory/Inventory.js

```diff
@@ -82,7 +82,8 @@ class Inventory extends Component {
     return (
       <div>
         <div className="m-3 col">
-          <ProductSearchSort />
+          {/* Comment this out. Search not working properly in Inventory Tab */}
```

Comment:
Should we actually comment it out? We could just disable it/not use it in demo.

Reply:
I just thought someone might ask if we could do a search there, so disabled it just to be safe.